LDAP-primer.md

```diff
@@ -248,6 +258,11 @@ The bind DN plays the role of username. The precise form of that DN depends on s
 configuration. It should be obvious that sending the password should only be done over
 a TLS-protected connection.
 
+Once a client has a connection, it may issue multiple operations to the server. The client
+can do this asynchronously, that is it does not have to wait for one operation to complete
+before issuing another operation. LDAP connections are often long lived, very unlike
+protocols such as HTTP.
+
```

Comment:
I don't think that the contrast with HTTP is so stark these days, since HTTP/2 and later encourage multiple requests per connection, and there are also websockets, so I'll leave out the last part.